hack/make/.integration-daemon-start

```diff
@@ -16,6 +16,10 @@ if [ "$(go env GOOS)" = 'windows' ]; then
        return
 fi
 
+if [ ! -x "/etc/docker/seccomp.json" ]; then
+	install_seccomp "$base/../../profiles/seccomp/default.json"
```

Comment:
@tianon PTAL this is a hack to make tests running - not sure where should this go honestly..